acumos-portal-fe/pom.xml

```diff
@@ -23,15 +23,15 @@ limitations under the License.
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.acumos.portal-marketplace</groupId>
 	<artifactId>acumos-portal-fe</artifactId>
-	<version>4.0.14-ai4eu-v3</version>
+	<version>4.0.14-ai4eu-v4</version>
 	<packaging>war</packaging>
 	<name>Acumos Portal frontend server</name>
 	<description>Acumos Portal AngularJS Front End</description>
 
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.16.RELEASE</version>
+		<version>2.1.1.RELEASE</version>
```

Comment:
why is it necessary to change to a newer spring boot framework?

Reply:
I mixed another pull request with this one. The updated spring and java is already in master.
It was partially updated by upstream, but the way they did it broke the ability to run it locally.
The updated Spring & Java has been running on acumos-dev until the environment was upgraded and no new issues were discovered (to my knowledge).

If it breaks, we can still revert it.

Reply:
Ah, then that's great!